apps/src/sites/studio/pages/levels/show.js

```diff
@@ -5,22 +5,24 @@ import {getStore, registerReducers} from '@cdo/apps/redux';
 import ScriptLevelRedirectDialog from '@cdo/apps/code-studio/components/ScriptLevelRedirectDialog';
 import UnversionedScriptRedirectDialog from '@cdo/apps/code-studio/components/UnversionedScriptRedirectDialog';
 import sectionData, {
-  setTtsAutoplayEnabled,
   setCodeReviewEnabled
 } from '@cdo/apps/redux/sectionDataRedux';
 import {setIsMiniView} from '@cdo/apps/code-studio/progressRedux';
+import instructions, {
+  setTtsAutoplayEnabledForLevel
+} from '@cdo/apps/redux/instructions';
 
 $(document).ready(initPage);
 
 function initPage() {
   const script = document.querySelector('script[data-level]');
   const config = JSON.parse(script.dataset.level);
 
+  registerReducers({sectionData, instructions});
   // this is the common js entry point for level pages
   // which is why ttsAutoplay is set here
-  registerReducers({sectionData});
   const ttsAutoplayEnabled = config.tts_autoplay_enabled;
-  getStore().dispatch(setTtsAutoplayEnabled(ttsAutoplayEnabled));
+  getStore().dispatch(setTtsAutoplayEnabledForLevel(ttsAutoplayEnabled));
```

Comment:
I'm not sure that the `forLevel` makes this clearer for me but don't feel strongly about this either way.

Reply:
What do you think would make it clearer for you? I want to distinguish between a section-level setting and the setting that is applied when the user visits the level